src/components/CourseEnrollments/index.jsx

```diff
@@ -3,6 +3,7 @@ import PropTypes from 'prop-types';
 import Helmet from 'react-helmet';
 import qs from 'query-string';
 import moment from 'moment';
+import _ from 'lodash';
```

Comment:
Should import just what we need from method packages, since we're only using 2 methods